### PR TITLE
[enterprise-4.13] HCIDOCS-125 Cherry Pick

### PR DESCRIPTION
**Pull request by kdrake20**
Cherry Picked from a4796056e3387de62c4fee81f6ae355cdff098be xref: https://github.com/openshift/openshift-docs/pull/70363

### CODE REVIEW

File: installing/installing_sno/install-sno-installing-sno.adoc [hunk 1]
```diff
@@ -72,6 +88,33 @@ include::modules/installation-aws_con_installing-sno-on-aws.adoc[leveloffset=+2]
 
 include::modules/install-sno-installing-with-usb-media.adoc[leveloffset=+1]
 
-include::modules/install-booting-from-an-iso-over-http-redfish.adoc[leveloffset=+1]
+[id="installing-sno-with-ibmpower"]
```
Comment (JoeAldinger) [inline installing/installing_sno/install-sno-installing-sno.adoc:91]: @kdrake20 it looks like you pulled in some additional content.

**Review by JoeAldinger — COMMENTED** [review]
I think you pulled in some additional content in the CP.

File: installing/installing_sno/install-sno-installing-sno.adoc (hunk 1 above)
Comment (kdrake20) [reply installing/installing_sno/install-sno-installing-sno.adoc]: Updated As Suggested.

**Comment by kdrake20** [comment]
Closing. See 4.13 cherry pick PR [here](https://github.com/openshift/openshift-docs/pull/71047).

**Pull request closed without merging**